test/test_operations_solve.jl

```diff
@@ -397,10 +397,7 @@ end
     parameters_value = [true, false]
     systems = [c_sys5, c_sys5_dc]
     networks = [DCPPowerModel, NFAPowerModel, StandardPTDFModel, CopperPlatePowerModel]
-    PTDF_ref = IdDict{System, PTDF}(
-        c_sys5 => build_PTDF5(),
-        c_sys5_dc => build_PTDF5_dc(),
-    )
+    PTDF_ref = IdDict{System, PTDF}(c_sys5 => build_PTDF5(), c_sys5_dc => build_PTDF5_dc())
```

Comment:
@daniel-thom made a fix in IS to avoid the issue with the UUID

Reply:
good. I don't think it really matters here, so I'll leave it as is unless anyone cares.